services/storage-users/pkg/command/uploads.go

```diff
@@ -69,7 +74,143 @@ func ListUploads(cfg *config.Config) *cli.Command {
 	}
 }
 
-// PurgeExpiredUploads is the entry point for the server command.
+// ListUploadSessions prints a list of upload sessiens
+func ListUploadSessions(cfg *config.Config) *cli.Command {
+	return &cli.Command{
+		Name:  "sessions",
+		Usage: "Print a list of upload sessions",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:        "id",
+				DefaultText: "unset",
+				Usage:       "filter sessions by upload session id",
+			},
+			&cli.BoolFlag{
+				Name:        "processing",
+				DefaultText: "unset",
+				Usage:       "filter sessions by processing status",
+			},
+			&cli.BoolFlag{
+				Name:        "expired",
+				DefaultText: "unset",
+				Usage:       "filter sessions by expired status",
+			},
+			&cli.StringFlag{
+				Name:  "output",
+				Usage: "output format to use (can be 'plain' or 'json', experimental)",
```

Comment:
Remove `experimental` we want to support this everywhere

Reply:
Also maybe a `--json` flag is enough? We don't need to support other formats imho.

Reply:
There might be report commands that dump csv or pdf. So, that is why I'd like to stick to a generic `--output` flag.

Reply:
I doubt that. Even if that would be special commands that ONLY output csv or pdf. So they have no need for an `--output` flag. 
I would strongly suggest to go with an easy to understand bool flag than to use a read-the-docs string flag. It will make live easier for users and support. 

But it's your PR. If you insist I can unhappily accept that. But remember my words when the first support tickets pop up.

Reply:
_read-the-docs_ is good point. I'll change to a `--json` flag.